app/services/plum_event_processor/null_processor.rb

```diff
@@ -0,0 +1,7 @@
+class PlumEventProcessor
+  class NullProcessor
+    def self.process
```

Comment:
Do we want to log these?  It could help catch thing that aren't processed correctly, or if the event stuff gets more complex in the future.